### ISSUE TITLE
issue with lcd screen flash

### ISSUE THREAD

**Issue by ElBruhchador** [rommulaner/Anycubic_Vyper_LCD_CE_6.1d#10]
after flashing the lcd screen and the MB firmware, i get a notification sayiing buiild is 622 but expected 61, and i dont know hhow to fix it

**Comment by rommulaner** [comment]
When it says it's expecting 61 and the screen is already 622, then the Mainboard is not flashed to 6.2.2.

**Comment by ElBruhchador** [comment]
ive tried to flash it to 6.2.2 but i dont think im doing it right or my printer just doesnt accept iit

**Comment by rommulaner** [comment]
Did you put the BIN only on a known working SD and have turned the printer on with the inserted card?

**Comment by ElBruhchador** [comment]
yes

**Comment by ElBruhchador** [comment]
if only ii had some sort of way to show you in real tiime

**Comment by rommulaner** [comment]
The SD needs to empty, only the BIN on the card. 
The message is correct?! It's not like expecting 622 but it's 61?
I think that the LCD was not flashed. Flash it again and make a picture of the End screen.

**Comment by ElBruhchador** [comment]
ii did everything but i think its my MB

**Comment by rommulaner** [comment]
Please flash the screen again and post a picture of the End screen. Then I can say more.

**Comment by ElBruhchador** [comment]
![Uploading 20240729_014854.png…]()

**Comment by ElBruhchador** [comment]
![20240729_014854](https://github.com/user-attachments/assets/08eb1515-aeef-47b2-ac23-2f145fe5f0b1)

**Comment by ElBruhchador** [comment]
sorry for the bad quality

**Comment by rommulaner** [comment]
Ok, LCD is flashed correctly. 
Perhaps you need to force to start the flashing process. 
Put the BIN on a working SD card with nothing else on it and put it in the running printer. Then tap long on the cube in the main menue until it jumps to the boot screen. Then tap long on lower edge of the screen until it starts flashing the Mainboard.
Done.

**Comment by ElBruhchador** [comment]
like insert the sd card, turn it on and then tap the cube on he main menu then spam tap the bottom to see if it works, and cube in the main as in once it boots up or while it boots up?

**Comment by ElBruhchador** [comment]
do you have discord or something?

**Comment by rommulaner** [comment]
No. Turn the printer on and you should get to main menu. Maybe it beeps the whole time. Then long tap and hold on the Anycubic Logo (sorry) for some seconds until it switches to the boot screen. Then insert the SD and tap and hold on the lower edge of the screen until it starts flashing.

**Comment by rommulaner** [comment]
You can write me on Facebook (https://www.facebook.com/Rommulaner) in the messenger or email me on rommulaner@gmail.com there I can give you my WhatsApp number.

**Comment by ElBruhchador** [comment]
i held it and i got somethging that says dev action:unkown

**Comment by rommulaner** [comment]
Hmm, what version did you have on the printer before? 6.1d?

**Comment by ElBruhchador** [comment]
yeah

**Comment by rommulaner** [comment]
Ok. You can flash the screen back to 6.1d and try this again or you connect a PC to the printer USB and send M997.

**Comment by rommulaner** [comment]
Have you tried to rename the BIN to something shorter with no spaces?

**Comment by rommulaner** [comment]
And make sure it's not the ZIP.

**Comment by ElBruhchador** [comment]
> Ok. You can flash the screen back to 6.1d and try this again or you connect a PC to the printer USB and send M997.

ive been trying this with pronterface bbut still wouldnt flash, and i havent renamed it and i have made sure its not the zip file

**Comment by ElBruhchador** [comment]
ok, i flashed the lcd back to 6.1D

**Comment by rommulaner** [comment]
Ok, now try to flash the MB to 6.2.2. If that doesn't work there is something wrong with your Mainboard.

**Comment by ElBruhchador** [comment]
i tried it but in the read me it says to have my lcd to 6.2.2 and i cant see the thing ive added

**Comment by ElBruhchador** [comment]
ima try to use cura to upload the file

**Comment by rommulaner** [comment]
I don't think it will work with cura, but you can try. It doesn't matter what you flash first.

**Comment by ElBruhchador** [comment]
> I don't think it will work with cura, but you can try. It doesn't matter what you flash first.
your right, it didnt work but just checking my board, i think a mofset chip burnt and damaghed surrounding areas

**Comment by EEdera** [comment]
Hi, I have the same issue as ElBruhchador. Same flashing screen as well. I did use a 32gb card (made a partition with 8gb fat32) and I put the .bin file on this new partition. Idk if I need to use a physical 8gb sd card for it? 

And also,  can I even use the LA_IS_7x7_FSD firmware, on my stock vyper without any mods? I am quite new to this type of modifications, and want to learn :)

**Comment by rommulaner** [comment]
Yes you can use them without any mods. FSD is better for stock setup.
What is the message exactly?
Not the other way around?
Flashing the MB is pretty easy and works nearly every time.

**Comment by EEdera** [comment]
Hi ty for your quick response :) 
build is 622 but expected 61

Idk if its because of the SD card, but I can gurantee that the Screen Upgrade worked 100%, I think its the MB that causes that error

**Comment by rommulaner** [comment]
No problem 😃
Yes, I think also the MB doesn't flash. You need to use a known working SD card and put only the bin file on it, nothing else. In my case also the micro SD from the screen worked with an adapter. Perhaps rename it and remove the LA_IS... could help.

**Comment by EEdera** [comment]
> No problem 😃
> Yes, I think also the MB doesn't flash. You need to use a known working SD card and put only the bin file on it, nothing else. In my case also the micro SD from the screen worked with an adapter. Perhaps rename it and remove the LA_IS... could help.

My bad, it said Flash your TFT Screen blablabla expected blablaba 

I will try

**Comment by EEdera** [comment]
The flash screen is different from yours

"download code files: 002" instead of 1 or 3 

This is weird

**Comment by rommulaner** [comment]
That doesn't matter as long as you copied the whole DWIN folder from the release onto the micro SD.

**Comment by EEdera** [comment]
> hen long tap and hold on the Anycubic Logo (sorry) for some seconds until it switches to the boot screen. Then insert the

If I try this, it says dev action: unknown

**Comment by EEdera** [comment]
exact the same issue as the guy that created this issue thread. I dont have any time anymore, today but did this work on other people´s printer?

**Comment by rommulaner** [comment]
You can try this. It's a hidden link to the boot screen where are two hidden buttons for reboot and firmware update. Try it you can't destroy anything.

**Comment by EEdera** [comment]
> You can try this. It's a hidden link to the boot screen where are two hidden buttons for reboot and firmware update. Try it you can't destroy anything.

Where is the hidden link?

**Comment by rommulaner** [comment]
It's written in the thread you quoted:

Turn the printer on and you should get to main menu. Maybe it beeps the whole time. Then long tap and hold on the Anycubic Logo (sorry) for some seconds until it switches to the boot screen. Then insert the SD and tap and hold on the lower edge of the screen until it starts flashing.

**Comment by EEdera** [comment]
> It's written in the thread you quoted:
> 
> Turn the printer on and you should get to main menu. Maybe it beeps the whole time. Then long tap and hold on the Anycubic Logo (sorry) for some seconds until it switches to the boot screen. Then insert the SD and tap and hold on the lower edge of the screen until it starts flashing.

If I try this I get the same error as the other guy,
dev action: unknown

**Comment by rommulaner** [comment]
Then flash the screen to version of the MB and try to flash the MB to the newest version with the hidden menu.

**Comment by ElBruhchador** [comment]
i have tried everything and nothing still works, i saw that someone used a 2gb sd card and it works but i only havea 8gb and it doesnt flash. idk what to keep trying

**Comment by EEdera** [comment]
> i have tried everything and nothing still works, i saw that someone used a 2gb sd card and it works but i only havea 8gb and it doesnt flash. idk what to keep trying

Try what roman wrote before, let me know if it worked.
Otherwhise I will try klipper, temporary

**Comment by rommulaner** [comment]
It's possible that the LCD Firmware isn't flashed to V3.5 although the BIN file is on the card. 
If it still says some different version during flashing, you need to add the additional OS bin files from the release an flash again.
Maybe that helps here.

**Comment by rommulaner** [comment]
Release updated for easier use and downgrade from 4.5 or higher versions.
